packages/picasso/src/PageSidebar/PageSidebar.tsx

```diff
@@ -92,6 +94,7 @@ export const PageSidebar = forwardRef<HTMLDivElement, Props>(function Sidebar(
 
   const handleCollapseButtonClick = useCallback(() => {
     setIsCollapsed(previousState => !previousState)
+    onCollapse?.()
```

Comment:
Looking at your use case - are you sure you don't want to have the current (or next) collapsed state being passed to this callback? If you need to have it sent to the backend, maybe you could benefit from something like this
```suggestion
    onCollapse?.(isCollapsed)
```

Reply:
Not really, in our case the initial value will be returned by BE and we will send to API the opposite value of it once talent collapse/extend the sidebar.

For our use case it's enough just to have a callback to fire when sidebar is changing.